content/index.md

```diff
@@ -45,11 +44,14 @@ tools:
 {% import 'quotes.macros.njk' as quotes %}
 {% import 'embed.macros.njk' as embed %}
 {% import 'layout.macros.njk' as layout %}
+{% import 'post.macros.njk' as post %}
 
 {{ layout.title('Featured Clients') }}
 
 {{ embed.logos(work, collections.all) }}
 
+{{ post.featured_posts('Article', collections) }}
```

Comment:
This is now "working", but is running into an issue similar to what we've found on Sass Site -- because this is a Markdown file, the HTML returned from this macro is then being parsed as Markdown, which we don't want. @mirisuzanne Suggestion for how to fix this? I suppose one option would be upgrading 11ty and then using `renderFile` to include this?

Reply:
Yeah, upgrading is not a bad idea anyway? The other alternative is making this an NJK file, and using markdown shortcodes where we want to keep markdown parsing.

Reply:
Makes sense. @oluoluoxenfree Is this something you'd like me to take a look at, or would you prefer to do it if I send some links/pointers? I'm fine either way.

Reply:
I'm happy for you to do it so I can read your PR/comments to see how you go about it! I'm never sure with stuff like that

Reply:
Fixed in #327.